qulice-checkstyle/src/main/java/com/qulice/checkstyle/MultiLineCommentCheck.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2011-2024 Qulice.com
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met: 1) Redistributions of source code must retain the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer. 2) Redistributions in binary form must reproduce the above
+ * copyright notice, this list of conditions and the following
+ * disclaimer in the documentation and/or other materials provided
+ * with the distribution. 3) Neither the name of the Qulice.com nor
+ * the names of its contributors may be used to endorse or promote
+ * products derived from this software without specific prior written
+ * permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+ * NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+ * FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+ * THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+ * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+ * OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.qulice.checkstyle;
+
+import com.puppycrawl.tools.checkstyle.api.AbstractCheck;
+import com.puppycrawl.tools.checkstyle.api.DetailAST;
+import com.puppycrawl.tools.checkstyle.api.TokenTypes;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Multi line comment checker.
+ * Used by the checkstyle process multiple times as a singleton.
+ * @since 0.23.1
+ */
+public final class MultiLineCommentCheck extends AbstractCheck {
+    /**
+     * Pattern for check.
+     * It is not final as it is initialized from the configuration.
+     */
+    private Pattern format = Pattern.compile("^$");
+
+    /**
+     * The message to report for a match.
+     * It is not final as it is initialized from the configuration.
+     */
+    private String message = "";
+
+    /**
+     * Comment line.
+     * It is not final because the visitToken method is called many times
+     * during the class under test and the field is reinitialized with a new object.
+     */
+    @SuppressWarnings("PMD.AvoidStringBufferField")
+    private StringBuilder line;
```

Comment:
@PeJetuz It seems, that the `line` is not an appropriate name of variable here. Since it's a "MultilineComment", I would give it `lines` or `text` name. What do you think?